### PR TITLE
Add license check to CI

### PR DESCRIPTION
**Pull request by Niederb**

### CODE REVIEW

File: about.hbs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            white-space: pre-wrap;
+        }
+    </style>
+</head>
+
+<body>
+    <main class="container">
+        <div class="intro">
+            <h1>Third Party Licenses</h1>
+            <p>This page lists the licenses of the projects used in cargo-about.</p>
```
Comment (clangenb) [inline about.hbs:42]: Here we should probably change 'cargo-about' to 'Integritee Networks Pallets' repository.

**Review by clangenb — COMMENTED** [review]
Looks good, cool. I just have one remark!

File: about.hbs (hunk 1 above)
Comment (Niederb) [reply about.hbs:42]: Yeah, probably... 🤣

**Review by clangenb — APPROVED** [review]
LGTM

**Pull request merged**